seqr/views/utils/orm_to_json_utils.py

```diff
@@ -213,28 +213,6 @@ def _get_json_for_sample(sample):
     return result
 
 
-def _get_json_for_dataset(dataset, add_sample_type_field=True):
```

Comment:
I feel like the fact you didn't need to make any changes to `_get_json_for_samples` is a sign we built this pretty well